docs/src/howto-guides/008_config_local_mqtt_port.md

```diff
@@ -34,14 +28,26 @@ tedge config get mqtt.port
 ```
 This prints out the port that has been set.
 
+## Step 3: Connect the thin edge device to cloud
+
+Using `tedge` command connect to the desired cloud as below.
+
+This will force all the services (mosquitto, tedge_mappers) to use newer port.
+
+```shell
+tedge connect c8y/az
+```
+
+Note: The step 1 and 2 can be followed in any order.
+
 ## Update to use default port
 
-To use the default port, one has to remove the `listener [port-number]` from the mosquitto.conf manually.
-Also has to unset the mqtt.port using the tedge cli framework as below
+To use the default port (1883), the mqtt.port has to be unset using the `tedge` command as below
 
 ```shell
 tedge config unset mqtt.port
 ```
+Once the port is reverted to default, the step 1 and 3 has to be followed to use the default port.
```

Comment:
Maybe you could provide links to sections with step 1 and step 3, there was one above where link to section could be useful. 
```suggestion
Once the port is reverted to default, the [step 1](#Step-3:-Connect-the-thin-edge-device-to-cloud) and 3 has to be followed to use the default port.
```

Reply:
fixed